packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-remote-media-item-node.js

```diff
@@ -255,110 +256,136 @@ export const createRemoteMediaItemNode = async ({
   const fetchState = {
     shouldBail: false,
   }
-  // Otherwise we need to download it
-  const remoteFileNode = await retry(
-    async () => {
-      if (fetchState.shouldBail) {
-        failedImageUrls.add(mediaItemUrl)
-        return null
-      }
 
-      const createFileNodeRequirements = {
-        parentNodeId: mediaItemNode.id,
-        store: gatsbyStore,
-        cache,
-        createNode,
-        createNodeId,
-        reporter,
-      }
+  const createFileNodeRequirements = {
+    parentNodeId: mediaItemNode.id,
+    store: gatsbyStore,
+    cache,
+    createNode,
+    createNodeId,
+    reporter,
+  }
 
-      if (hardCacheMediaFiles) {
-        // check for file in .wordpress-cache/wp-content
-        // if it exists, use that to create a node from instead of
-        // fetching from wp
-        try {
-          const buffer = await fs.readFile(hardCachedFilePath)
-          const node = await createFileNodeFromBuffer({
-            buffer,
-            name: title,
-            ext: path.extname(mediaItemUrl),
-            ...createFileNodeRequirements,
-          })
-
-          if (node) {
-            return node
-          }
-        } catch (e) {
-          // ignore errors, we'll download the image below if it doesn't exist
-        }
-      }
+  let imageNode
 
-      const { hostname: wpUrlHostname } = url.parse(wpUrl)
-      const { hostname: mediaItemHostname } = url.parse(mediaItemUrl)
+  const shouldFetchImage = fileSize > maxFileSizeBytes
 
-      const htaccessCredentials = pluginOptions.auth.htaccess
+  // if this file is larger than maxFileSizeBytes, don't fetch the remote file
+  if (shouldFetchImage) {
+    reporter.warn(
+      formatLogMessage(
+        `Remote media item not fetched because its size exceeded the maxFileSizeBytes config option: ${mediaItemNode?.mediaDetails?.file}`
+      )
+    )
+
+    imageNode = await createNode({
```

Comment:
@TylerBarnes Is there anywhere else we should pass this node to `touchNode` to avoid garbage collection on build updates?